### PR TITLE
Small update to scheduler benchmark

### PR DESCRIPTION
**Pull request by gmarek**
cc @hongchaodeng

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/34237)

### CODE REVIEW

File: test/integration/scheduler_perf/scheduler_test.go [hunk 1]
```diff
@@ -18,31 +18,46 @@ package benchmark
 
 import (
 	"fmt"
+	"math"
 	"testing"
 	"time"
 )
 
+const (
+	threshold3K  = 30
```
Comment (wojtek-t) [inline test/integration/scheduler_perf/scheduler_test.go]: Can we make it higher?

File: test/integration/scheduler_perf/scheduler_test.go [hunk 2]
```diff
@@ -51,16 +66,23 @@ func schedulePods(numNodes, numPods int) {
...
 		// This can potentially affect performance of scheduler, since List() is done under mutex.
 		// Listing 10000 pods is an expensive operation, so running it frequently may impact scheduler.
 		// TODO: Setup watch on apiserver and wait until all pods scheduled.
 		scheduled := schedulerConfigFactory.ScheduledPodLister.Indexer.List()
-		fmt.Printf("%ds\trate: %d\ttotal: %d\n", time.Since(start)/time.Second, len(scheduled)-prev, len(scheduled))
 		if len(scheduled) >= numPods {
-			return
+			return minQps
+		}
+		// There's no point of printing it for the last iteration, as the value is random
```
Comment (wojtek-t) [inline test/integration/scheduler_perf/scheduler_test.go]: s/of/in/ ?

Comment (gmarek) [inline test/integration/scheduler_perf/scheduler_test.go]: Done.

File: test/integration/scheduler_perf/scheduler_test.go (hunk 1 above)
Comment (gmarek) [inline test/integration/scheduler_perf/scheduler_test.go]: Made it 100.

File: test/integration/scheduler_perf/scheduler_test.go [hunk 3]
```diff
@@ -18,31 +18,60 @@ package benchmark
...
 	}
-	schedulePods(1000, 30000)
+	if min := schedulePods(1000, 30000); min < threshold30K {
+		t.Errorf("To small pod scheduling throughput for 30k pods. Expected %v got %v", threshold30K, min)
+	} else {
+		fmt.Printf("Minimal observed throughput for 30k pod test: %v\n", min)
+	}
 }
 
+// TestSchedule1000Node30KPods schedules 30k pods on 1000 nodes.
```
Comment (wojtek-t) [inline test/integration/scheduler_perf/scheduler_test.go]: 2000Node60k

Comment (gmarek) [inline test/integration/scheduler_perf/scheduler_test.go]: Ups...

**Comment by wojtek-t** [comment]
lgtm

**Pull request merged**